docbook/Developers_Guide/en-US/Events_Reference_Output.xml

```diff
@@ -412,11 +438,17 @@
 			<blockquote>
 				<para>
 					This event allows plugins to output HTML code immediately after the line of an attachment.
+					Recives an attachment, the array returned by the file_get_visible_attachments() function.
 				</para>
 
 				<itemizedlist>
+					<title>Parameters</title>
+					<listitem><para>&lt;Array&gt;: the attachment data</para></listitem>
```

Comment:
Maybe better to put the text from line 441 above (i.e. *the array returned by the file_get_visible_attachments() function*) here